test/testLenClamp.mo

```diff
@@ -0,0 +1,36 @@
+import List "mo:base/List";
+import Trie "mo:base/Trie";
+import Debug "mo:base/Debug";
+
+type List<T> = List.List<T>;
+
+/* copied because private to Trie.mo */
```

Comment:
Would be nice to be able to access private fields for tests only

Reply:
Yes, this is a common problem with testing oo stuff. Not sure how to fix that.